project/CLI.py

```diff
@@ -55,6 +55,8 @@ def logout(self, _):
 @need_admin
 def add_team(self, args):
     try:
+        if self.game is None:
```

Comment:
Just info (no change needed) instead of doing an if here you could instead add another except
```Python
except AttributeError:
    return sc.team_add_fail
```